Unix/provreg/provreg.c

```diff
@@ -528,7 +528,7 @@ static int _AddEntry(
          */
         if (strcmp(hosting, PROV_REG_HOSTING_INPROC) == 0)
         {
-            e->hosting = PROV_HOSTING_INPROC;
+            e->hosting = PROV_HOSTING_REQUESTOR;
```

Comment:
Yuck. Why not just make INPROC be invalid, and return an error? I see no reason why asking, explicitly, for INPROC should not be an error and would do something completely different ...

Please explain your rationale, thanks.